### PR TITLE
Add express consistency test

### PR DESCRIPTION
**Pull request by letalumil**
Add a test to reproduce issues described in #26.

### CODE REVIEW

**Comment by letalumil** [comment]
Add a possible solution to the problem described.

File: index.js [hunk 1]
```diff
@@ -59,8 +59,11 @@ function wrap(fn, isParam) {
     // Ensure next function is only ran once
     arguments[2 + isErrorHandler] = _once(arguments[2 + isErrorHandler]);
     try {
-      await fn.apply(null, arguments);
-      arguments[1 + isErrorHandler].headersSent ? null : arguments[2 + isErrorHandler]();
+      const promise = fn.apply(null, arguments);
+      if (promise && typeof promise.then === 'function' && promise[Symbol.toStringTag] === 'Promise') {
```
Comment (vkarpov15) [inline index.js:63]: I'd remove the `toStringTag` check, most apps just check `promise && typeof promise.then === 'function'` to check if `promise` is a [thenable](https://masteringjs.io/tutorials/fundamentals/thenable)

**Review by vkarpov15 — CHANGES_REQUESTED** [review]
Thanks. One quick comment, but otherwise LGTM. We'll have to ship 0.8.0 for this since it might be backwards breaking.

File: index.js (hunk 1 above)
Comment (letalumil) [reply index.js]: Thanks, fixed!

**Review by vkarpov15 — APPROVED** [review]

**Pull request merged**